src/libs/Navigation/types.ts

```diff
@@ -366,8 +367,10 @@ type PublicScreensParamList = {
     [SCREENS.TRANSITION_BETWEEN_APPS]: {
         shouldForceLogin: string;
         email: string;
+        error: string;
         shortLivedAuthToken: string;
-        exitTo: string;
+        shortLivedToken: string;
+        exitTo: Routes;
```

Comment:
Please check that all of the fields are really should be required, `error` for example I guess can be optional